cla_public/static-src/stylesheets/main.scss

```diff
@@ -389,3 +389,31 @@ dl.govuk-list.govuk-list--bullet {
 .laa-flee-button-container {
   display:none;
 }
+
+body:not(.cla-production) {
+  .govuk-header__container {
+    border-bottom-color:govuk-colour("orange");
+  }
+  .govuk-phase-banner__content__tag{
+    background-color:govuk-colour("orange");
+  }
+}
+
+body.cla-staging {
+  .govuk-header__container {
+    border-bottom-color:govuk-colour("pink");
+  }
+  .govuk-phase-banner__content__tag {
+    background-color:govuk-colour("pink");
+  }
+}
+
+body.cla-dev {
+  .govuk-header__container {
+    border-bottom-color:govuk-colour("turquoise");
+  }
+  .govuk-phase-banner__content__tag{
+    background-color:govuk-colour("turquoise");
+  }
+}
```

Comment:
Can you change these to use SCSS variables?  
Change the `govuk-colour("orange")` etc to:
- `$laa-unknown-colour`
- `$laa-staging-colour`
- `$laa-dev-colour`

Then at the top of `main.scss`, add the following to line 26:
```
$laa-unknown-colour: govuk-colour("orange");
$laa-staging-colour: govuk-colour("turquoise");
$laa-dev-colour: govuk-colour("pink");